changelogs/unreleased/gh-4693-formats-for-spaceless-tuples.md

```diff
@@ -0,0 +1,4 @@
+## feature/box
+
+* Added support of formats for tuple created via `box.tuple.new`, i.e,
+  spaceless tuples (gh-4693).
```

Comment:
```suggestion
* Introduced `box.tuple.format` that enables format definition for tuples
  created via `box.tuple.new`(spaceless tuples) (gh-4693).
```

Reply:
Applied your suggestion, thanks!

Reply:
I added another changelog considering the breaking change of `box.tuple.new`, please review.